### PR TITLE
[CHI-215] Add scrolling example to modal component docs

### PR DESCRIPTION
**Pull request by davidojedacl**
@mattnickles @jllr

### CODE REVIEW

File: src/website/views/components/modal.pug [hunk 1]
```diff
@@ -224,3 +224,64 @@ p.-text Remove <code>a-modal__header</code> and <code>a-modal__footer</code> for
...
+
+.example.-mb--3
+  .a-backdrop.-p--6.-position--relative.-z--0(style="border-top-left-radius:3px;border-top-right-radius:3px;")
+    .a-modal(style="margin-top:0;")
+      .a-modal__header
+        h3.a-modal__title Modal Title
+        button(class='a-btn -icon -close')
+          .a-btn__content
+            i.a-icon.icon-x
+      .a-modal__content(style='max-height:150px;')
```
Comment (mattnickles) [inline src/website/views/components/modal.pug]: Change to `max-height: 400px;`

File: src/website/views/components/modal.pug [hunk 2]
```diff
@@ -224,3 +224,64 @@ p.-text Remove <code>a-modal__header</code> and <code>a-modal__footer</code> for
...
+      <div class="a-modal">
+        <div class="a-modal__header">
+          <h3 class="a-modal__title">Modal Title</h3>
+          <button class="a-btn -icon -close">
+            <div class="a-btn__content">
+              <i class="a-icon icon-x"></i>
+            </div>
+          </button>
+        </div>
+        <div class="a-modal__content" style="max-height:150px">
```
Comment (mattnickles) [inline src/website/views/components/modal.pug]: Change to `max-height: 400px;`

File: src/website/views/components/modal.pug [hunk 3]
```diff
@@ -224,3 +224,64 @@ p.-text Remove <code>a-modal__header</code> and <code>a-modal__footer</code> for
...
+
+.example.-mb--3
+  .a-backdrop.-p--6.-position--relative.-z--0(style="border-top-left-radius:3px;border-top-right-radius:3px;")
+    .a-modal(style="margin-top:0;")
+      .a-modal__header
+        h3.a-modal__title Modal Title
+        button(class='a-btn -icon -close')
+          .a-btn__content
+            i.a-icon.icon-x
+      .a-modal__content(style='max-height:400px;')
```
Comment (mattnickles) [inline src/website/views/components/modal.pug]: Max-height was increased but we need more content to trigger the overflow-y. Can you add an extra lorem ipsum paragraph? 5 or 6 paragraphs should do it with a -mt--0 on the first-child.

File: src/website/views/components/modal.pug [hunk 4]
```diff
@@ -224,3 +224,64 @@ p.-text Remove <code>a-modal__header</code> and <code>a-modal__footer</code> for
         </div>
       </div>
     </div>
+
+h4 Scrollable
```
Comment (mattnickles) [inline src/website/views/components/modal.pug]: Can you move this Scrollable modal example above the simple modal example?

File: src/website/views/components/modal.pug [hunk 5]
```diff
@@ -224,3 +224,64 @@ p.-text Remove <code>a-modal__header</code> and <code>a-modal__footer</code> for
...
+p.-text
+  | If the content height is bigger than the content modal container
+  | height, we can make the content scrollable.
+h5 Requirements
+p.-text
+  | Content Modal must have the<code>max-height</code> defined.
```
Comment (mattnickles) [inline src/website/views/components/modal.pug]: Let's add two new utility sizing classes and encourage devs to use those versus adding inline styles. `-mh--400` and `-mh--480`

File: src/website/views/components/modal.pug (hunk 3 above)
Comment (davidojedacl) [reply src/website/views/components/modal.pug]: ok

File: src/website/views/components/modal.pug (hunk 4 above)
Comment (davidojedacl) [reply src/website/views/components/modal.pug]: done

File: src/website/views/components/modal.pug (hunk 5 above)
Comment (davidojedacl) [reply src/website/views/components/modal.pug]: done

File: src/website/views/components/modal.pug [hunk 6]
```diff
@@ -43,6 +43,78 @@ h4 Base
       </div>
     </div>
 
+h4 Scrollable
+p.-text
+  | If the content height is bigger than the content modal container
```
Comment (mattnickles) [inline src/website/views/components/modal.pug]: Change to:
Enable scrolling if the height of the modal's content is larger than the modal's content container.

File: src/website/views/components/modal.pug [hunk 7]
```diff
@@ -43,6 +43,78 @@ h4 Base
...
+  | height, we can make the content scrollable.
+h5 Requirements
+p.-text
+  | Content Modal must have the<code>max-height</code> defined.
```
Comment (mattnickles) [inline src/website/views/components/modal.pug]: Change to:

`A <code>max-height</code> must be defined on <code>a-modal__content</code>. Use max-height utility classes such as <code>-mh--400</code> or <code>-mh--480</code>, or define your own value.`

File: src/website/views/components/modal.pug (hunk 6 above)
Comment (davidojedacl) [reply src/website/views/components/modal.pug]: done

File: src/website/views/components/modal.pug (hunk 7 above)
Comment (davidojedacl) [reply src/website/views/components/modal.pug]: done

**Pull request merged**